pkgs/applications/audio/fluidsynth/default.nix

```diff
@@ -14,7 +14,7 @@ stdenv.mkDerivation rec {
     sha256 = "0x5808d03ym23np17nl8gfbkx3c4y3d7jyyr2222wn2prswbb6x3";
   };
 
-  nativeBuildInputs = [ pkg-config cmake ];
+  nativeBuildInputs = [ buildPackages.stdenv.cc pkg-config cmake ];
```

Comment:
Should this be in depsBuildBuild, since the target platform is also the build platform? (Does it matter?)